src/components/RequestInfo.tsx

```diff
@@ -0,0 +1,317 @@
+                                        />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </ContentSection>
+            }
+        >
+            <ContentSection parentClassName="!my-0 !py-0 !pb-8 lg:hidden" className="!my-0 !py-0 ">
+                <div className="flex flex-col lg:hidden">
+                    <SocialProofSection iconItems={SocialProofIcons} />
+                    <CaseStudySection />
+                </div>
+            </ContentSection>
+
+            <ExpectationsSection items={expectationItems} />
+            <LinksCardSection />
+            <div className="sg-demo-gradient h-44 w-full" />
+        </Layout>
+    )
+}
+
+interface HeaderListTabProps {
+    contents: { icon: ReactNode; header: string; listItems: string[] }[]
+}
+
+const HeaderListTab: FunctionComponent<HeaderListTabProps> = ({ contents }) => (
+    <div className="flex flex-col gap-10">
+        {contents.map(content => (
+            <div key={content.header} className="flex flex-row gap-x-4">
+                <div>{content.icon}</div>
+                <div className="flex flex-col gap-y-2">
+                    <div className="text-xl font-semibold leading-[26px] tracking-tight text-gray-700">
+                        {content.header}
+                    </div>
+                    <ul className="ml-0 list-inside pl-3 text-lg leading-[27px] text-gray-500">
+                        {content.listItems.map(listItem => (
+                            <li key={listItem}>{listItem}</li>
+                        ))}
+                    </ul>
+                </div>
+            </div>
+        ))}
+    </div>
+)
+
+interface SocialProofIconsProps {
+    iconItems: {
+        id: string
+        src: string
+        className: string
+    }[]
+}
+
+const SocialProofSection: FunctionComponent<SocialProofIconsProps> = ({ iconItems }) => (
+    <div>
+        <div className="font-mono text-lg leading-[27px] tracking-normal text-[#5E6E8C]">
+            Trusted by dev teams at leading companies
+        </div>
+        <div className="mt-6 flex flex-row flex-wrap justify-center gap-8 lg:justify-start">
+            {iconItems.map(icon => (
+                <img key={icon.id} src={icon.src} alt={`${icon.id}`} className={icon.className} />
+            ))}
+        </div>
+    </div>
+)
+
+interface ExpectationsSectionProps {
+    items: {
+        id: number | string
+        label: string
+    }[]
+}
+
+const ExpectationsSection: FunctionComponent<ExpectationsSectionProps> = ({ items }) => (
+    <ContentSection parentClassName="!pb-16 lg:!pb-24 !pt-8 lg:!pt-16">
+        <div className="flex flex-col gap-16 lg:items-center">
+            <h2 className="text-left text-[32px] leading-[38.4px] !tracking-tighter text-gray-700 lg:max-w-[400px] lg:max-w-full lg:text-center">
```

Comment:
We shouldn't need all this inline CSS styling when we have the H2 global styling setup with definitions for text size, leading, and tracking. Please remove. Also, please make sure to check all the other headings on the page and remove inline css styling where it isn't needed. This ensures we can make global updates in the future.